elements/rh-button/rh-button.css

```diff
@@ -0,0 +1,391 @@
+  --_danger-border-color: transparent;
+  --_danger-active-color: var(--rh-color-text-primary-on-dark, #ffffff);
+  --_danger-active-background-color: #a30000; /* WARNING: non-token value */
+  --_danger-active-border-color: transparent;
+  --_danger-focus-color: var(--rh-color-text-primary-on-dark, #ffffff);
+  --_danger-focus-background-color: #a30000;  /* WARNING: non-token value */
+  --_danger-focus-border-color: transparent;
+  --_danger-hover-color: var(--rh-color-text-primary-on-dark, #ffffff);
+  --_danger-hover-background-color: #a30000; /* WARNING: non-token value */
+  --_danger-hover-border-color: transparent;
+
+  /* SECONDARY */
+  --_secondary-color: var(--rh-color-interactive-blue-darker, #0066cc);
+  --_secondary-background-color: transparent;
+  --_secondary-border-color: var(--rh-color-interactive-blue-darker, #0066cc);
+  --_secondary-active-color: var(--rh-color-interactive-blue-darker, #0066cc);
+  --_secondary-active-background-color: transparent;
+  --_secondary-active-border-color: var(--rh-color-interactive-blue-darker, #0066cc);
+  --_secondary-focus-color: var(--rh-color-interactive-blue-darker, #0066cc);
+  --_secondary-focus-background-color: transparent;
+  --_secondary-focus-border-color: var(--rh-color-interactive-blue-darker, #0066cc);
+  --_secondary-hover-color: var(--rh-color-interactive-blue-darker, #0066cc);
+  --_secondary-hover-background-color: transparent;
+  --_secondary-hover-border-color: var(--rh-color-interactive-blue-darker, #0066cc);
+
+  /* TERTIARY */
+  --_tertiary-color: var(--rh-color-text-primary-on-light, #151515);
+  --_tertiary-background-color: transparent;
+  --_tertiary-border-color: var(--rh-color-border-strong-on-light, #151515);
+  --_tertiary-active-color: var(--rh-color-text-primary-on-light, #151515);
+  --_tertiary-active-background-color: transparent;
+  --_tertiary-active-border-color: var(--rh-color-border-strong-on-light, #151515);
+  --_tertiary-focus-color: var(--rh-color-text-primary-on-light, #151515);
+  --_tertiary-focus-background-color: transparent;
+  --_tertiary-focus-border-color: var(--rh-color-border-strong-on-light, #151515);
+  --_tertiary-hover-color: var(--rh-color-text-primary-on-light, #151515);
+  --_tertiary-hover-background-color: transparent;
+  --_tertiary-hover-border-color: var(--rh-color-border-strong-on-light, #151515);
+
+  /* LINK */
+  --_link-color: var(--rh-color-interactive-blue-darker, #0066cc);
+  --_link-background-color: transparent;
+  --_link-active-color: #a30000; /* WARNING: non-token value */
+  --_link-active-background-color: transparent;
+  --_link-focus-color: var(--rh-color-interactive-blue-darkest, #004080);
+  --_link-focus-background-color: transparent;
+  --_link-hover-color: var(--rh-color-interactive-blue-darkest, #004080);
+  --_link-hover-background-color: transparent;
+
+  /* CLOSE */
+  --_close-color: var(--rh-context-light-color-text-muted, #6a6e73);
+  --_close-background-color: transparent;
```

Comment:
Hover:
`--_close-hover-color: var(--rh-color-black-900, #151515);`
`--_close-hover-background-color: transparent;`

Focus:
`--_close-focus-color: var(--rh-color-black-900, #151515);`
`--_close-focus-background-color: transparent;`
Blue border around the whole button is correct.

Active:
`--_close-active-color: var(--rh-color-black-900, #151515);`
`--_close-active-background-color: transparent;`